README.md

```
@@ -6,31 +6,71 @@ You can read about the technical specifications of the Ootle in the [RFCs](https
+rustup target add wasm32-unknown-unknown
 cargo run --bin tari_swarm_daemon --release -- -c data/swarm/config.toml init
-# Edit your config. You may need to point it to the path for the tari L1 repo. By default it assumed it's checked out at `../tari`.
 cargo run --bin tari_swarm_daemon --release -- -c data/swarm/config.toml start
 ```
 
-This will start a Minotari base node, a Minotari console wallet, an Ootle validator node, a wallet and an indexer.
-Additionally, it will automatically submit the validator node registration and mine blocks until the validator node is
-active.
+> Note: For subsequent runs, you only need to run the third command with the `-k` argument to avoid trying to re-register the Validator Nodes: `cargo run --bin tari_swarm_daemon --release -- -c data/swarm/config.toml start -k`
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)